client/src/components/ProfileMenu.js

```diff
@@ -40,9 +44,73 @@ const ProfileMenu = (props) => {
     props.history.push("/");
   };
 
+  const uploadProfilePicture = async () => {
+    const auth = await (await fetch("/auth/status")).json();
+    const userId = auth.data.user_id;
```

Comment:
I think this is already available from the updated user Context that Adams added